src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task03.java

```diff
@@ -35,8 +35,23 @@ public static void main(String[] args) {
      * @return null/'RED'/'ORANGE'/'YELLOW'/'GREEN'
      */
     static String getColorHealthPoint(int currentHealthPoint, int maxHealthPoint) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
+        final double red = 0.25;
+        final double orange = 0.5;
+        final double yellow = 0.75;
```

Comment:
"Violation: 'NAME_CONSTANT' was not used for constant naming
Solution: Rename constant"